<launch>
    <node pkg="dvl_a50_ros_driver" type="publisher" name="publisher">
        <param name="ip" value="192.168.194.95"/>
        <param name="port" value="16171"/>
        <param name="dvl_topic" value="/dvl/velocity"/>
        <param name="dead_reckoning_topic" value="/dvl/dead_reckoning"/>
    </node>
</launch>
